test/hub-values.yaml

```diff
@@ -0,0 +1,42 @@
+dash:
```

Comment:
Related to my comment below, but would be nice to eliminate the need for this file as it's the same as `examples/hub-values.yaml` minus the templating differences